docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3936,13 +3936,13 @@ This rule will be applied to `example.org` and subdomains pages and will execute
 
 **Exceptions**
 
-Exception rules can disable some scriptlets on particullar domains.
+Exception rules can disable some scriptlets on particullar domains. The syntax for exception scriptlet rules is similar to normal scriptlet rules but using `#@%#` instead of `#%#`:
```

Comment:
```suggestion
Exception rules can disable some scriptlets on particullar domains. The syntax for exception scriptlet rules is similar to normal scriptlet rules but uses `#@%#` instead of `#%#`: